test/e2e/helpers.js

```diff
@@ -760,36 +760,9 @@ const sendScreenToConfirmScreen = async (
   await openActionMenuAndStartSendFlow(driver);
   await driver.fill('[data-testid="ens-input"]', recipientAddress);
   await driver.fill('.unit-input__input', quantity);
-  if (process.env.MULTICHAIN) {
-    // check if element exists and click it
-    await driver
-      .findElement({
-        text: 'I understand',
-        tag: 'button',
-      })
-      .then(
-        (_found) => {
-          driver.clickElement({
-            text: 'I understand',
-            tag: 'button',
-          });
-        },
-        (error) => {
-          console.error('Element not found.', error);
-        },
-      );
```

Comment:
I don't think that this should be removed. The toolTipApproveButton label is used in the [SendPageContent component](https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/multichain/pages/send/components/content.tsx#L83-L103), that is used on the new send page.